config.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+type skipConfig struct {
+	Packages map[string]map[string]bool `json:"packages"`
+}
+
+func readSkipConfig(file string) (_ *skipConfig, err error) {
```

Comment:
My feeling is that making a custom encoding like this is just a recipe for foot guns and reduces the options the Deployments team has to generate these files. We should standardize on JSON or bring in a YAML parser.

Reply:
ah, i thought that it was going to be hand edited in configcat. if it's auto generated then yeah I'll go back to JSON

Reply:
Yeah, I'm not sure what approach they are going to take, but they can have a simplified representation in that layer that they convert down to something standard on our side if they want. JSON on this side seems safes and more universal.

Thoughts @wojciechka?

Reply:
BTW: I do like the list instead of maps down in the file, too. Duplicates shouldn't be much of a problem and can just be handled gravely when loading it in.

Reply:
How might you skip a whole package's tests?

Reply:
There's no way to do that currently, but it would be trivial to add support for. Do we have a use case for doing that right now - it seems like quite a blunt instrument?

Reply:
Yeah seems okay to leave out. They can decide whether or not they need it. 👍🏻

Reply:
I agree, lists would be more readable, especially if we move it to YAML.

Reply:
done